README.md

```diff
@@ -138,6 +138,22 @@ make
 sudo make install
 ```
 
+## Upgrade
+
+If you previously installed Cog from a GitHub Releases URL, you can upgrade to the latest version by running the same commands you used to install it:
+
+```console
+sudo curl -o /usr/local/bin/cog -L "https://github.com/replicate/cog/releases/latest/download/cog_$(uname -s)_$(uname -m)"
+sudo chmod +x /usr/local/bin/cog
+```
+
+If you're using macOS and you previously installed Cog with Homebrew, run the following:
+
+```console
+brew update
+brew install cog
```

Comment:
If you already have it installed, I'm pretty sure this should be `brew upgrade cog`.
```console
❯ brew upgrade cog
Warning: Treating cog as a formula. For the cask, use homebrew/cask/cog
Warning: cog 0.8.6 already installed
❯ brew install cog
Warning: Treating cog as a formula. For the cask, use homebrew/cask/cog
Warning: cog 0.8.6 is already installed and up-to-date.
To reinstall 0.8.6, run:
  brew reinstall cog
```

Reply:
Correct 👍🏼